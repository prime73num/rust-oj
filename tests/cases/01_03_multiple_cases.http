###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 1186
# content-type: application/json
# date: Sat, 03 Sep 2022 05:07:57 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Accepted","time":0},{"id":2,"info":"","memory":0,"result":"Accepted","time":0},{"id":3,"info":"","memory":0,"result":"Accepted","time":0},{"id":4,"info":"","memory":0,"result":"Accepted","time":0},{"id":5,"info":"","memory":0,"result":"Accepted","time":0},{"id":6,"info":"","memory":0,"result":"Accepted","time":0},{"id":7,"info":"","memory":0,"result":"Accepted","time":0},{"id":8,"info":"","memory":0,"result":"Accepted","time":0},{"id":9,"info":"","memory":0,"result":"Accepted","time":0},{"id":10,"info":"","memory":0,"result":"Accepted","time":0}],"created_time":"2022-09-03T05:07:58.016Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-03T05:07:58.016Z"}
