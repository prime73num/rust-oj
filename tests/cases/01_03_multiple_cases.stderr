[2022-09-03T04:02:06Z INFO  actix_server::builder] Starting 2 workers
[2022-09-03T04:02:06Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-03T04:02:07Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000295
[2022-09-03T04:02:07Z INFO  post_jobs_handler] Post jobs
[2022-09-03T04:02:08Z INFO  post_jobs_handler] job run success
[2022-09-03T04:02:08Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 1186 "-" "-" 0.438120
